src/clj/game/cards/assets.clj

```diff
@@ -979,6 +979,17 @@
                                                    (trigger-event state side :no-trash card))))}
                                card targets))}}
 
+   "NGO Front"
+   {:advanceable :always
+    :abilities [{:advance-counter-cost 1
+                 :effect (effect (trash card) (gain :credit 5))
+                 :label "Gain 5 [Credits]"
```

Comment:
Let's put `[Trash]` at the beginning of the label string, to show the trashcan icon in the menu item itself: `"[Trash]: Gain 5 [Credits]"`.

Reply:
I'd be very tempted to make both these abilities into an ability-maker function that takes advance-counter-cost and credit-gain to reduce the minor repetition.